underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -595,7 +597,11 @@ protected String getRootKey() {
   @Override
   protected InputStream openObject(String key, OpenOptions options) throws IOException {
     try {
-      return new S3AInputStream(mBucketName, key, mClient, options.getOffset());
+      RetryPolicy retryPolicy = new ExponentialBackoffRetry(
```

Comment:
can we call ObjectUnderFileSystem.getRetryPolicy or call ObjecUnderFileSystem.retryOnException here?